bin/magic-script.js

```diff
@@ -56,16 +56,19 @@ require('yargs') // eslint-disable-line
   .command('build [target]', 'Compile project', yargs => {
     yargs.option('debug', {
       alias: 'd',
+      describe: 'build debug version of the app',
       boolean: true,
       default: true
     });
     yargs.option('install', {
       alias: 'i',
+      describe: 'install on the device after build',
       boolean: true,
       default: false
     });
     yargs.option('host', {
       alias: 'h',
+      describe: 'host to build on',
```

Comment:
This isn't specifying the host, it's specifying that we should build for lumin on desktop.